Formula/g/git-sync.rb

```diff
@@ -1,6 +1,6 @@
 class GitSync < Formula
   desc "Clones a git repository and keeps it synchronized with the upstream"
-  homepage "https://github.com/kubernetes/git-sync#readme"
```

Comment:
@hezhizhen thanks for the PR, can you elaborate about this change?

Reply:
It's just removing unnecessary suffix.